packages/fxa-auth-server/fxa-oauth-server/lib/config.js

```diff
@@ -240,28 +240,51 @@ const conf = convict({
     },
   },
   openid: {
+    // See ../docs/signing-key-management.md to read more about what these different keys are for.
     keyFile: {
-      doc: 'Path to Private key JWK to sign id_tokens',
+      doc: 'Path to private key JWK to sign various kinds of JWT tokens',
       format: String,
       default: '',
       env: 'FXA_OPENID_KEYFILE',
     },
+    newKeyFile: {
+      doc:
+        'Path to private key JWK that will be used to sign JWTs in the future',
+      format: String,
+      default: '',
```

Comment:
With the default of `''`, I can't run `NODE_ENV=dev node ./prepare-new-signing-key.js`

```
scripts git:(scripted-key-rotation) NODE_ENV=dev node ./prepare-new-signing-key.js                                                                                            git:(scripted-key-rotation|)
assert.js:339
    throw err;
    ^

AssertionError [ERR_ASSERTION]: openid.newKeyFile not specified
    at main (/Users/shanetomlinson/development/fxa/packages/fxa-auth-server/fxa-oauth-server/scripts/prepare-new-signing-key.js:28:5)
    at Object.<anonymous> (/Users/shanetomlinson/development/fxa/packages/fxa-auth-server/fxa-oauth-server/scripts/prepare-new-signing-key.js:51:3)
    at Module._compile (internal/modules/cjs/loader.js:776:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Function.Module.runMain (internal/modules/cjs/loader.js:829:12)
    at startup (internal/bootstrap/node.js:283:19)
    at bootstrapNodeJSCore (internal/bootstrap/node.js:622:3)
```

Maybe in dev.json, set a name for oldKeyFile and newKeyFile so these can be run locally w/o needing to edit anything.